README.md

```diff
@@ -74,6 +74,42 @@ Edools::Enrollment.update(id: 1, school_product_id: 75506, status: "active")
 Edools::Enrollment.all
 ```
 
+### LessonProgress example
```

Comment:
Please add the expected return format at the readme as the openstruct can be a bit surprising.